README.md

```diff
@@ -51,27 +51,16 @@ v1.0 - Intial release for iOS with full support of all operations based on plan
 
 ```shell
 npm install --save react-native-sqlite-storage
+rnpm link
```

Comment:
Is rnpm something that comes bundled with react-native-cli or needs to be downloaded and installed separately? If separately, we need some instructions on how to get it installed before we tell users to run the command.